dashboard/app/models/level_group.rb

```diff
@@ -49,7 +49,7 @@ def levels
       end
     end
 
-    Level.where(name: level_names)
+    Level.where(name: level_names).order_as_specified(name: level_names)
```

Comment:
I'm not sure we need the gem unless we need to chain additional queries onto the returned levels.  Could this just be:
```ruby
Level.where(name: level_names).sort_by{ |l| level_names.index(l.name) }
```